lib/puppet/provider/base_dsc/powershell.rb

```diff
@@ -76,6 +76,13 @@ def munge_boolean(value)
     raise ArgumentError.new "invalid value: #{value}"
   end
 
+  def munge_ensure(value)
+    return 'absent' if value.downcase =~ (/^(absent|disable)$/i)
+    return 'present' if value.downcase =~ (/^(present|enable)$/i)
```

Comment:
Likewise, shouldn't this look at the value that the type has defined for it's `present_value`?